MapleServer2/Types/BuddyEmote.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using MaplePacketLib2.Tools;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using MapleServer2.Tools;
+
+namespace MapleServer2.Types
+{
+    public class BuddyEmote
```

Comment:
I'm a little confused by this class. Is it just related to buddy emotes or it's an early buddy list implementation?

Reply:
Sorry, this was me experimenting in the beginning. This should be deleted.